### ISSUE TITLE
[v1][Request] Add isActive reactive property to LoadingBar

### ISSUE THREAD

**Issue by rossity** [quasarframework/quasar#4071]
This property already exists on the `Loading` plugin and would allow you to do things like this

```
<q-btn :loading="$q.LoadingBar.isActive()" label="Submit Form"></q-btn>
```

Another possible but separate request would be a reactive property of the current progress of the loading bar.

```
<q-btn :loading="$q.LoadingBar.isActive()" :percentage="Math.floor($q.LoadingBar.progess() * 100)" label="Submit Form"></q-btn>
```

This could greatly help in areas where the user might need to see an additional visual indicator of a background process or ajax request happening.

**Comment by smolinari** [comment]
@rossity

You've added isActive methods in your examples. 😄 

I'm not sure of the purpose of `isActive` inside `Loading`. But, theoretically, you'd need logic where you "show" or "hide" the loading activity for the `Loading` plugin. That same sort of logic is also needed to "start" and "stop" the loading bar. And that logic should just as easily have the logic you need for the button loading prop. In other words, it's stead of wiring the loading facilities in sequence through reactive props the plugins, you should be wiring the loading facilities in parallel i.e over Vuex or a global event bus.

I hope that makes sense. 

Scott

**Comment by rossity** [comment]
@smolinari a couple things

There are two plugins:

1. [Loading](https://v1.quasar-framework.org/quasar-plugins/loading) - which has the `isActive` reactive prop already
2. [LoadingBar](https://v1.quasar-framework.org/quasar-plugins/loading-bar) - which does not

LoadingBar's main purpose, and default function, is to provide ease of use surrounding the [QAjaxBar](https://v1.quasar-framework.org/vue-components/ajax-bar) component. QAjaxBar's by default to quote the docs `captures Ajax calls automatically (unless told not to).`

So there would be no additional logic needed to start and stop the loading bar, which is why it would be nice to have this additional prop. Because then you can add the button examples above at the end of a form and have the button show a loading indicator as well as the LoadingBar indicator at the top of the page when you submit it. No need for Vuex or a Global Event bus or additional logic.

**Comment by rstoenescu** [comment]
Will be available in beta.23.

**Comment by smolinari** [comment]
@rossity - You learn something new every day. 😄 I didn't realize LoadingBar watches for AJAX calls automatically. 

Scott

**Comment by StefanoMantero** [comment]
Is the `isActive` prop usable now? I can't find a way to correctly listen to the isActive prop of the global component, or just `start` and `stop` events without actually instance the component!  😭
